db/dcrpg/upgrades.go

```diff
@@ -602,10 +646,6 @@ func (pgb *ChainDB) handleUpgrades(client *rpcutils.BlockGate,
 			return false, fmt.Errorf("failed to index agendas table: %v", err)
 		}
 
-		log.Infof("Index the agendas table on Block Time...")
```

Comment:
Might as well remove the above `IndexAgendasTableOnAgendaID` too since the agendas table is just going to be dropped and recreated.

Reply:
`IndexAgendasTableOnAgendaID` is still in use to create the unique constraint for the agendas table.

Reply:
I meant this particular call to `IndexAgendasTableOnAgendaID`.  A subsequent upgrade just drops the agendas table, correct?  Then after that it is indexed.